ZupIT/ritchie-cli
# adding legatsy pipeline to allow running old code

**- What I did**
I added things that are needed to run pipeline using our legacy code

**- How to verify it**
See if while trying to run pipeline using branches named like `legacy-.*` if no errors occur.
